reference/6/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md

```diff
@@ -0,0 +1,350 @@
+"powershell".
+
+```Schema
+"LogIdentity": "<log-identity>"
+```
+
+Where:
+
+- `<log-identity>` is the string identity that PowerShell should use
+  for writing to syslog.
+
+> [!NOTE]
+> You may want to have different **LogIdentity** values for each different
+> instance of PowerShell you have installed.
+
+In this example, we are configuring the **LogIdentity** for a preview
+release of PowerShell.
+
+```json
+{
+  "LogIdentity": "powershell-preview"
+}
+```
+
+### LogLevel
+
+> [!IMPORTANT]
+> This setting can only be used in macOS and Linux.
+
+Specifies the minimum severity level at which PowerShell should log.
+
+```Schema
+"LogLevel": "<log-level>|default"
+```
+
+Where:
+
+- `<log-level>` is one of:
+  - Always
+  - Critical
+  - Error
+  - Warning
+  - Informational
+  - Verbose
+  - Debug
+
+> [!NOTE]
+> Setting a the log level enables all log levels above it.
+
+Setting this setting to **default** will be interpreted as the default value.
+The default value is **Informational**.
+
+The following example sets the value to **Verbose**.
+
+```json
+{
+  "LogLevel": "Verbose"
+}
+```
+
+### LogChannels
+
+> [!IMPORTANT]
+> This setting can only be used in macOS and Linux.
+
+Determines which logging channels are enabled.
+
+```Schema
+"LogChannels": "<log-channel>,..."
+```
+
+Where:
+
+- `<log-channel>` is one of:
+  - Operational - logs basic information about PowerShell activities
+  - Analytic - logs more detailed diagnostic information
+
+The default value is **Operational**. To enable both channels, include both
+values as a single comma-separated string. For example:
+
+```json
+{
+  "LogChannels": "Operational,Analytic"
+}
+```
+
+### LogKeywords
+
+> [!IMPORTANT]
+> This setting can only be used in macOS and Linux.
+
+Determines which parts of PowerShell are logged. By default, all log keywords are enabled.
+To enable multiple keywords, list the values in a single comma-separated string.
+```Schema
+"LogKeywords": "<log-keyword>,..."
+```
+
+Where:
+
+- `<log-keyword>` is one of:
+  - Runspace - runspace management
+  - Pipeline - pipeline operations
+  - Protocol - communication protocol handling, such as PSRP
+  - Transport - transport layer support, such as SSH
+  - Host - PowerShell host functionality, for example console interaction
+  - Cmdlets -PowerShell builtin cmdlets
+  - Serializer - serialization logic
+  - Session - PowerShell session state
+  - ManagedPlugin - WSMan plugin
+
+> [!NOTE]
+> It is generally advised to leave this value unset
+> unless you are trying to diagnose a specific behavior
+> in a known part of PowerShell.
+> Changing this value only decreases the amount of information logged.
+
+This example limits the logging to runspace operations, pipeline logic, and
+cmdlet use. All other logging will be omitted.
+
+```json
+"LogKeywords": "Runspace,Pipeline,Cmdlets"
+```
+
+<!--
+## Group policy configuration
+
+### ScriptExecution
+
+### ScriptBlockLogging
```

Comment:
This is definietly all platforms
```
{
  "PowerShellPolicies": {
    "ScriptBlockLogging": {
      "EnableScriptBlockLogging": true,
      "EnableScriptBlockInvocationLogging": true
    }
  }
}
```

Reply:
`EnableScriptBlockLogging` logs script block creation
`EnableScriptBlockInvocationLogging` logs script block execution start and stop